Makefile

```diff
@@ -142,6 +142,25 @@ frontend-lint: node_modules/.uptodate
 bddtests: python
 	@tox -qe bddtests
 
+PIP_COMPILE := pyenv exec pip-compile
+
+requirements/%.txt: requirements/%.in
+	$(PIP_COMPILE) --allow-unsafe --quiet $(PIP_COMPILE_FLAGS) $<
+
+requirements/dev.txt: requirements/requirements.txt
+requirements/tests.txt: requirements/requirements.txt
+requirements/functests.txt: requirements/requirements.txt
+requirements/bddtests.txt: requirements/requirements.txt
+requirements/lint.txt: requirements/tests.txt requirements/functests.txt requirements/bddtests.txt
+
+pip_compile_executable := $(shell pyenv root)/versions/$(shell pyenv version-name)/bin/pip-compile
```

Comment:
The template will actually be able to make this a bit faster by removing the need to shell out to `pyenv root` and `pyenv version-name`